gestion_base_adresse/install/sql/adresse/10_FUNCTION.sql

```diff
@@ -32,16 +33,17 @@ CREATE FUNCTION adresse.calcul_num_adr(pgeom public.geometry) RETURNS TABLE(num
     test boolean;
 BEGIN
 
-    SELECT adresse.calcul_point_position(adresse.calcul_segment_proche(geom, pgeom),pgeom)into isleft
-    FROM( SELECT geom, id_voie, ST_Distance(pgeom, geom) as dist
-    FROM adresse.voie
-    WHERE statut_voie_num IS FALSE ORDER BY dist LIMIT 1) AS d;
-
+    -- Get idvoie
     SELECT id_voie into idvoie
     FROM( SELECT id_voie, ST_Distance(pgeom, geom) as dist
     FROM adresse.voie
     WHERE statut_voie_num IS FALSE ORDER BY dist LIMIT 1) AS d;
 
+    SELECT adresse.calcul_point_position(adresse.calcul_segment_proche(geom, pgeom),pgeom)into isleft
+    FROM( SELECT geom, id_voie, ST_Distance(pgeom, geom) as dist
+    FROM adresse.voie
+    WHERE statut_voie_num IS FALSE AND id_voie=idvoie ORDER BY dist) AS d;
```

Comment:
```suggestion
    SELECT adresse.calcul_point_position(adresse.calcul_segment_proche(geom, pgeom),pgeom ) into isleft
    FROM adresse.voie
    WHERE statut_voie_num IS FALSE AND id_voie=idvoie;
```
Ceci devrait être suffisant